homeassistant/components/axis/hub/api.py

```diff
@@ -29,6 +35,7 @@ async def get_axis_api(
             port=config[CONF_PORT],
             username=config[CONF_USERNAME],
             password=config[CONF_PASSWORD],
+            web_proto=config.get(CONF_PROTOCOL, "http"),
```

Comment:
May be this should become a config option as well so we can suggest `https` as the default protocol?

Reply:
No I don't like to expose this kind of config over options, if it works it works and they don't need to change anything. If they want to limit to https traffic on device it will trigger a reconfiguration flow